packages/material-ui-lab/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "5.0.0-alpha.22",
   "private": false,
   "author": "Material-UI Team",
-  "description": "Material-UI Lab - Incubator for Material-UI React components.",
+  "description": "Laboratory incubator for Material-UI.",
```

Comment:
"Lab" ( for experiments), or "incubator" (for growing and hatching things)?

Reply:
```suggestion
  "description": "Laboratory for new Material-UI modules.",
```